packages/core-components/src/layout/Header/Header.tsx

```diff
@@ -34,26 +36,26 @@ const useStyles = makeStyles<BackstageTheme>(theme => ({
     display: 'flex',
     flexDirection: 'row',
     flexWrap: 'wrap',
-    justifyContent: 'flex-end',
     alignItems: 'center',
     backgroundImage: theme.page.backgroundImage,
     backgroundPosition: 'center',
     backgroundSize: 'cover',
   },
   leftItemsBox: {
+    maxWidth: '100%',
     flex: '1 1 auto',
+    marginBottom: theme.spacing(1),
   },
   rightItemsBox: {
     flex: '0 1 auto',
     display: 'flex',
     flexDirection: 'row',
     flexWrap: 'wrap',
     alignItems: 'center',
-    marginRight: theme.spacing(1),
+    width: 'auto',
```

Comment:
Let's maybe not ship redundant code and add unnecessary bundle size: 

![image](https://user-images.githubusercontent.com/9947422/125911571-2f2a58c3-c72a-4841-a7ad-50cfb7aae797.png)

If this overrides something inherited from a higher level in the cascade, maybe we can remove that value instead of adding `auto` here?

Reply:
Thanks for mentioning it! In general I would totally agree with you here, but which is not really obvious from the codes is, that I'm overriding styles enforced by MUI because as this is a `Grid` container displaying the actions. A alternative could be `width: initial`, which would enforce the initial `auto` default value.

<img width="190" alt="Screenshot 2021-07-16 at 15 06 52" src="https://user-images.githubusercontent.com/8904624/125952641-d90b4ff2-8150-4013-8585-5a2882852993.png">